docs/updates/migrate-v3.md

```diff
@@ -16,18 +16,30 @@ Going forward, **only projects with Specification Versions 2.0 and higher are su
 In case a legacy specification version is detected, **an automatic migration is attempted.**
 This means your old projects might still work. Unless they have non-standard configuration in their ui5.yaml.
 
+## Changes for Projects
+
+!!! info
+    ✅ Projects defining **Specification Version 2.x** are expected to be **fully compatible with UI5 Tooling v3**
+
+For projects defining the latest **Specification Versions 3.0 and higher**, some changes apply:
+
+* **Breaking Change:** The `metadata.name` property is now restricted to contain only certain characters and no uppercase letters. See [Configuration: `name`](../pages/Configuration.md#name) for details
+
+Also see [Configuration: Specification Version 3.0](../pages/Configuration.md#specification-version-30-beta).
+
 ## Changes for Extensions
 
 !!! info
-    ✅ Custom Tasks and -Middleware defining **Specification Version 2.x and lower** are expected to be **fully compatible with UI5 Tooling v3**
+    ✅ Custom Tasks and -Middleware defining **Specification Version 2.x** are expected to be **fully compatible with UI5 Tooling v3**
```

Comment:
--> "Custom Tasks and Custom Middlewares"

Reply:
Done